docs/conf.py

```diff
@@ -20,7 +20,7 @@
 # - Add Raiden path to the path for sphinx to import
 import os
 import sys
-sys.path.insert(0, os.path.dirname(os.path.abspath('.')))
```

Comment:
It doesn't matter, you can run any of the

`cd docs && make html
make -C docs html
make docs`

the result is the same, with -C option make will change its working dir.